### PR TITLE
Attempting to rephrase why ads are bad.

### PR DESCRIPTION
**Pull request by Ipstenu**
Please note: THIS IS NOT COMPLETE YET.

The reason it’s a pull request is to make it easier for us to see what was changed and discuss. As of April 2019, it is NOT considered ready. There will be a public call for comment on make.wordpress.org/plugins when we feel it’s at a good state.

### CODE REVIEW

**Comment by nicomollet** [comment]
Maybe rephrase `Advertising is only permitted by Automattic`.

Seriously, changing the rules to allow your own advertizing is more than borderline.
There will be consequences.

File: guideline-07.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <h4>7. Plugins may not track users without their consent.</h4>
 
-In the interest of protecting user privacy, plugins may not contact external servers without <em>explicit</em> and authorized consent. This is commonly done via an 'opt in' method, requiring registration with a service or a checkbox within the plugin settings. Documentation on how any user data is collected, and used, should be included in the plugin’s readme, preferably with a clearly stated privacy policy.
+In the interest of protecting user privacy, plugins may not contact external servers without <em>explicit</em> and authorized consent. This is commonly done via an 'opt in' method, requiring registration with a service, or a checkbox within the plugin settings. Documentation on how any user data is collected, and used, must be included in the plugin’s readme, with a link to a clearly stated privacy policy and/or terms of us.
```
Comment (JPry) [inline guideline-07.md]: I believe there is a small typo: It appears this should be "terms of use"

**Comment by javorszky** [comment]
One more question: because these are advertisement guidelines, and they are now permitted, what are the guidelines surrounding tracking of these ads? Impressions / sites / actions taken (dismissed (which really is just snoozing)), so that we can calculate ROI.

Can we implement A/B testing here?

**Comment by JPry** [comment]
> they are now permitted

@javorszky They've always been permitted. This isn't new. "Discouraged" doesn't mean "not permitted".

**Comment by nerrad** [comment]
My observations:

- As pointed out by @JPry there's no change in whether advertisements are permitted or not.  All that has changed is that there is the dropping of the use "discouraged".
- I find the wording more concise and clearer.  Takeaways: advertisements are permitted, they must be for services or products directly related to the developer or plugin installed, third party advertisement systems are not allowed.
- The clarification on third party ad systems is much clearer and direct.  I think thats a significant _good_ change.  The original wording had some ambiguity around that.

On the whole, I _like_ these changes.  Thanks!

**Comment by Ipstenu** [comment]
Hey everyone, thanks for jumping in.

This is NOT READY FOR PRIME TIME YET. I updated the description to explain that.

This is MY idea, and mine alone. I’ve had the code on my laptop for a while, and made a pull so it could be looked at more easily as a whole.

NOTHING WILL BE CHANGED WITHOUT A PUBLIC POST ON MAKE/PLUGINS

Please. Calm down. The goal was to make things easier for some ESL developers who have difficulty understanding that SHOULD and MAY have different connotations. Hence saying ‘not permitted’ instead of MAY remove access.

**Comment by Ipstenu** [comment]
> And what's with the change in guideline-03? Changing `may result in a plugin being removed.` to `is not permitted.` looks like a weaker statement. What if a plugin dev decides to host somewhere else, what are the consequences, now that a plugin clearly cannot be removed based on this..

You read that backwards. MAY is the weaker sentence. By saying something IS NOT PERMITTED we are saying you CANNOT do it. And since we already say at the top that any violation may result in being removed, it’s redundant.

**Comment by Ipstenu** [comment]
> One more question: because these are advertisement guidelines, and they are now permitted, what are the guidelines surrounding tracking of these ads? Impressions / sites / actions taken (dismissed (which really is just snoozing)), so that we can calculate ROI.
> 
> Can we implement A/B testing here?

Nothing has changed YET. This is NOT LIVE. Good lord, it’s like you’ve never seen a pull request b

**Comment by Ipstenu** [comment]
I’m canceling this. Y’all jumped in before anything was remotely ready,

File: guideline-07.md (hunk 1 above)
Comment (Ipstenu) [reply guideline-07.md]: Yes! Thank you. Patched in for the next edit :)

**Comment by Ipstenu** [comment]
See https://github.com/WordPress/wporg-plugin-guidelines/pull/66 for a more complete, and slightly closer to prime-time pull request.
